src/utils/utilities.jl

```diff
@@ -210,6 +210,48 @@ function prettytime(t)
     return @sprintf("%.3f %s", value, units)
 end
 
+import Dates
+using Dates
```

Comment:
Do we need both?

Reply:
I'll ditch the `using` and just qualify things